front/src/modules/timesStops/styles/_timesStopsDatasheet.scss

```
@@ -1,31 +1,79 @@
 .time-stops-datasheet {
+  padding-bottom: 16px;
+  border-radius: 10px;
+  box-shadow:
+    0px 1px 0px rgba(255, 255, 255, 1) inset,
+    0px 4px 7px -3px rgba(255, 171, 88, 0.17),
+    0px 2px 4px rgba(0, 0, 0, 0.22);
+  background-color: var(--ambiantB15);
+  --dsg-cell-background-color: var(--white100);
+  --dsg-cell-disabled-background-color: var(--white100);
+  --dsg-header-text-color: var(--grey50);
+  --dsg-header-active-text-color: var(--black100);
+
   .activeRow {
-    font-weight: bold;
-    background-color: var(--coolgray1);
-    --dsg-cell-background-color: var(--coolgray1);
-    --dsg-cell-disabled-background-color: var(--coolgray1);
-    .dsg-cell {
-      font-weight: bold;
+    font-weight: semibold;
```

With `semibold`, I don't see any change :/ I think you can use 600 for now, we will have to add new vars in the css
```suggestion
    font-weight: 600;
```

---

> Ah hum it's still 50px

The last batch of changes isn't pushed yet

> With semibold, I don't see any change

sure